### ISSUE TITLE
Synchronize exif with website

### ISSUE THREAD

**Issue by jidanni** [commons-app/apps-android-commons#5135]
In Preferences there's choices of which exif fields one should send. However the default on the official website upload form is much wider. Therefore some fields will be chopped off using the app's default. The two should be synchronized so there won't be any difference, in the default state.

I'm saying the app should be shipped with more liberal defaults.

**Comment by whym** [comment]
Some quick thoughts: I can see it both ways - more consistency with the website vs more protection of privacy (and in Android, I think Google tries to move us towards the latter). Giving a choice post-install, and maybe reminding at some point later too, might be better.

For example, we can give two options of 1) fully open (or as open as the website) and 2) a limited set of metadata (current default), telling that the user can change it later.

I think there can be reasons to have different defaults, though. On PC, many use retouch software during which the user can sanitize metadata in the way they prefer. On phone, I think the same cannot be said, not to the same extent anyway.

**Comment by nicolas-raoul** [comment]
I totally agree with Whym that desktop and Android are different, with very different privacy expectations.

In order to keep the app simple to use for non-technical users, I am in favor of **not** showing post-install settings suggestions, though.

@jidanni When using the website all EXIF fields are kept, right?

**Comment by whym** [comment]
A related phabricator issue on the desktop side: [Determine workflow to selectively purge potentially privacy-sensitive EXIF fields, such as geocoordinates, from a Wikimedia Commons file](https://phabricator.wikimedia.org/T218057)

**Comment by jidanni** [comment]
All I know is I got different results using the website versus using the app. I'm sure the people on the website have made some policy, as they usually do. So you might as well use the same policy.

**Comment by whym** [comment]
I could be wrong but it seems like the website basically preserves all the metadata submitted, except for clearly errorneous data. `makeFilteredData()` seems to fix some errors, but not remove not fundamentally change in metadata fields: https://github.com/wikimedia/mediawiki/blob/8dbf403b1a7b9e951bbd5aba05a804eb8ad94225/includes/media/Exif.php

**Comment by jidanni** [comment]
You might say if their picture includes their mother's shoes and she's picky about it, well then they should aim their camera better.  The same thing with the exif stuff. I mean certainly they agreed to some terms when they began using the app.
